base/applications/regedit/childwnd.c

```diff
@@ -25,8 +12,8 @@
 
 ChildWnd* g_pChildWnd;
 static int last_split;
-HBITMAP SizingPattern = 0;
-HBRUSH  SizingBrush = 0;
+HBITMAP SizingPattern;
+HBRUSH  SizingBrush;
```

Comment:
Are these changes fixing something?

Reply:
no that doesn't fix anything, it just makes the code shorter.

Reply:
Then avoid these sort of changes.

Reply:
Shorter code is better code.